src/ansys/dpf/core/workflow.py

```diff
@@ -659,12 +662,12 @@ def get_recorded_workflow(id, server=None):
         return wf
 
     @property
-    def info(self):
+    def info(self) -> dict[str, list[str]]:
```

Comment:
You could use [TypedDict](https://peps.python.org/pep-0589/) if the list of keys is fixed.